metadata-ingestion/src/datahub/ingestion/source/sql/sql_config.py

```diff
@@ -91,7 +91,7 @@ def get_sql_alchemy_url(self):
         pass
 
 
-class BasicSQLAlchemyConfig(SQLAlchemyConfig):
+class SQLAlchemyConnectionConfig(ConfigModel):
```

Comment:
we should map out what we want the full hierarchy to look like in the future

Reply:
Yeah...